test/smoke/behavior-mixin.html

```diff
@@ -22,6 +22,7 @@
   </template>
 </dom-module>
 <script>
+  /** @polymerBehavior */
```

Comment:
Oh, this slipped in from some other investigation. It's an improvement though so /shruggie